src/redux/greetingSlice.js

```diff
@@ -0,0 +1,49 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  value: '',
+  status: 'idle',
+};
+
+export const fetchRandomGreeting = createAsyncThunk(
+  'greetings/fetchRandomGreeting',
+  async () => {
+    try {
+      const response = await axios('http://127.0.0.1:3000/api/greetings/random');
+      if (!response) {
+        throw new Error('Can not fetch greetings');
+      }
+      const data = await response.data;
+
+      return data.greeting;
```

Comment:
- [ ] As required by the previous CR please 🙏🏽 ensure that all the greetings are rendered, to fix this issue the correct line should be `return data` since you don't have a key in the JSON object just values, check this example 👇🏽 

| return data.greeting | return data |
|---|---|
| `{ greeting: 'Hello', }`|`{ 'Hello', }`|